documentation/slate/source/includes/apis.md

```diff
@@ -964,20 +964,23 @@ The [isochrones](#isochrones) service exposes another response structure, which
 
 ### Precisions on `forbidden_uris[]` and `allowed_id[]`
 
-These parameters are filtering the vehicle journeys and the stop points used to compute the journeys. The journeys can only use allowed vehicle journeys (as present in the `public_transport` or `on_demand_transport` sections). They also can only use the allowed stop points (as present in the `street_network`, `waiting` and `crow_fly` sections).
+These parameters are filtering the vehicle journeys and the stop points used to compute the journeys.
+The principle is to create only a blacklist using them (no whitelist is ever managed).
```

Comment:
This does not seems very straighforward.

And I don't get the whitelist part (we **do** manage whitelist)